.bazelrc

```diff
@@ -35,35 +37,34 @@ build:build_cpp --test_tag_filters=+allwpilib-build-cpp --build_tag_filters=+all
 build:no_example --test_tag_filters=-wpi-example --build_tag_filters=-wpi-example
 test:no_example --test_tag_filters=-wpi-example --build_tag_filters=-wpi-example
 
-# Artifactory Cache Setup
-build:base_remote --remote_timeout=3600
-
-# Additional suggestions from buildbuddy for speed
-build:base_remote --experimental_remote_cache_compression
-build:base_remote --experimental_remote_cache_compression_threshold=100
-build:base_remote --noslim_profile
-build:base_remote --experimental_profile_include_target_label
-build:base_remote --experimental_profile_include_primary_output
-build:base_remote --nolegacy_important_outputs
-
-common:base_remote_readonly --noremote_upload_local_results
-build:artifactory_readonly --config=base_remote_readonly
-build:artifactory_readonly --remote_cache=https://frcmaven.wpi.edu/artifactory/wpilib-generic-cache-bazel-local
-# workaround for https://github.com/bazelbuild/bazel/issues/17700
-build:artifactory_readonly --remote_cache_header=X-JFrog-Download-Redirect-To=None
-
 # Build Buddy Cache Setup
 build:build_buddy --bes_results_url=https://app.buildbuddy.io/invocation/
 build:build_buddy --bes_backend=grpcs://remote.buildbuddy.io
 build:build_buddy --remote_cache=grpcs://remote.buildbuddy.io
-build:build_buddy --config=base_remote
+build:build_buddy --remote_timeout=3600
+
+# Additional suggestions from buildbuddy for speed
+build:build_buddy --experimental_remote_cache_compression
+build:build_buddy --experimental_remote_cache_compression_threshold=100
+build:build_buddy --noslim_profile
+build:build_buddy --experimental_profile_include_target_label
+build:build_buddy --experimental_profile_include_primary_output
+build:build_buddy --nolegacy_important_outputs
+
+common:build_buddy_readonly --noremote_upload_local_results
 
 # This config should be used locally. It downloads more than the CI version
-build:remote_user --config=base_remote
-build:remote_user --config=artifactory_readonly
+build:remote_user --config=build_buddy
+build:remote_user --config=build_buddy_readonly
 build:remote_user --remote_download_toplevel
 
-build:ci --config=base_remote
+build:ci --config=build_buddy
 build:ci --remote_download_minimal
 
 build --build_metadata=REPO_URL=https://github.com/wpilibsuite/allwpilib.git
+
+common --define="WPILIB_VERSION=2025.424242.3.1-unknown"
```

Comment:
Hopefully [this ](https://github.com/bazel-contrib/rules_jvm_external/issues/1414) can make our lives easier, but in the meantime can this be a placeholder version (like `0.0.0-local` or something along thos lines, and then add something like this to the CI command line action `--define=WPILIB_VERSION="$(git describe --dirty)"`

A quick local test shows that cmd line arguments override bazelrc arguments. This way you can always have something there for the make variable substitution, but then use a better value when you need to.

Reply:
@ThadHouse , what do you recommend here?

Reply:
Happy to do a follow-on change when we've got a better idea of what we want.  For now, let's leave it as is.  Would love insights from Thad or someone else when they are ready.